src/main/java/com/java_essential/varycheva/essential/homework/hw1/task1/Rectangle.java

```diff
@@ -0,0 +1,32 @@
+package com.java_essential.varycheva.essential.homework.hw1.task1;
+
+public class Rectangle {
+
+    private double side1;
+    private double side2;
+
+    public Rectangle(double side1, double side2){
+        if (side1 <= 0 || side2 <= 0){
+            throw new IllegalArgumentException("Сторона должна быть положительным числом");
+        } else {
+            this.side1 = side1;
+            this.side2 = side2;
+        }
```

Comment:
You are using your own code - in this case it is better to check this data at a higher level.
Check it out where you scan.

Reply:
fixed